Proposing alternative solution to DCF

linkhp opened the pull request.

linkhp committed:
Proposing alternative solution to DCF

msmimart commented on articles/active-directory/external-identities/google-federation.md:
```diff
@@ -86,6 +86,11 @@ Modify your apps to use the system browser for sign-in. For details, see [Embedd
 Before Google puts these changes into place on September 30, 2021, Microsoft will deploy a workaround for apps still using embedded web-views to ensure that authentication isn't blocked. Users who sign in with a Gmail account in an embedded web-view will be prompted to enter a code in a separate browser to finish signing in.
 
+Alternatively, you can have your existing and new Gmail users sign in with email one-time passcode. To have your Gmail users use email one-time passcode:
+1. [Enable email one-time passcode](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/one-time-passcode#enable-email-one-time-passcode)
```
```suggestion
1. [Enable email one-time passcode](one-time-passcode.md#enable-email-one-time-passcode)
```

msmimart commented on articles/active-directory/external-identities/google-federation.md:
```diff
@@ -86,6 +86,11 @@ Modify your apps to use the system browser for sign-in. For details, see [Embedd
 
+Alternatively, you can have your existing and new Gmail users sign in with email one-time passcode. To have your Gmail users use email one-time passcode:
+1. [Enable email one-time passcode](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/one-time-passcode#enable-email-one-time-passcode)
+2. [Remove Google Federation](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/google-federation#how-do-i-remove-google-federation)
```
```suggestion
2. [Remove Google Federation](google-federation.md#how-do-i-remove-google-federation)
```

msmimart commented on articles/active-directory/external-identities/google-federation.md:
```diff
@@ -86,6 +86,11 @@ Modify your apps to use the system browser for sign-in. For details, see [Embedd
+Alternatively, you can have your existing and new Gmail users sign in with email one-time passcode. To have your Gmail users use email one-time passcode:
+1. [Enable email one-time passcode](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/one-time-passcode#enable-email-one-time-passcode)
+2. [Remove Google Federation](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/google-federation#how-do-i-remove-google-federation)
+3. [Reset redemption status](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/reset-redemption-status) of your Gmail users so they can use email one-time passcode instead.
```
```suggestion
3. [Reset redemption status](reset-redemption-status.md) of your Gmail users so they can use email one-time passcode instead.
```

msmimart reviewed (commented):
@linkhp looks good, we just need to use relative link format for these internal docs. Thanks!

linkhp committed:
Update google-federation.md

"Microsoft apps on Windows" is affected, so removing this from "change does not affect" list and moving to "known scenarios that will impact Gmail users" by adding this: - Microsoft apps (e.g. Teams and PowerApps) on Windows

linkhp committed:
Update articles/active-directory/external-identities/google-federation.md

Co-authored-by: Michele Martin <36167252+msmimart@users.noreply.github.com>

linkhp committed:
Update articles/active-directory/external-identities/google-federation.md

Co-authored-by: Michele Martin <36167252+msmimart@users.noreply.github.com>

linkhp committed:
Update articles/active-directory/external-identities/google-federation.md

Co-authored-by: Michele Martin <36167252+msmimart@users.noreply.github.com>

linkhp committed:
Update google-federation.md
